src/modules/shared/pages/notifications/notifications-list.component.html

```diff
@@ -93,15 +93,18 @@ <h2 class="nhsuk-heading-xs nhsuk-u-margin-0">Filters</h2>
                 <span class="nhsuk-table-responsive__heading">{{ notificationsList.getColumnLabel('notification') }}</span>
 
                 <ng-container *ngIf="!item.link">
-                  {{ 'shared.catalog.innovation.notification_context_details.' + item.contextDetail + '.title' | translate: item.params }}
+                  <span *ngIf="item.readAt !== null" class="nhsuk-table-responsive__text">
+                    {{ 'shared.catalog.innovation.notification_context_details.' + item.contextDetail + '.title' | translate: item.params }}
+                  </span>
+
                   <a *ngIf="item.readAt === null" href="javascript:void(0)" (click)="onNotificationClick($event, item.id, null)">
-                    Mark as read.
+                    {{ 'shared.catalog.innovation.notification_context_details.' + item.contextDetail + '.title' | translate: item.params }}
                     <span class="nhsuk-u-visually-hidden">Click to mark as read</span>
                   </a>
                 </ng-container>
```

Comment:
Keep how it was (the lines in red, please). This changed due to modifications about notifications without links.